lib/wallet/wallet.js

```diff
@@ -1716,10 +1773,11 @@ class Wallet extends EventEmitter {
    */
 
   async _createOpen(name, force, options) {
-    const acct = options ? options.account || 0 : 0;
-    const mtx = await this.makeOpen(name, force, acct);
-    await this.fill(mtx, options);
-    return this.finalize(mtx, options);
+    const {mtx, errors, isAllError} = await this._createBatchOpen(Array.of(name), force, options);
+    if (isAllError)
+      throw new Error(errors[0].error);
+    else
```

Comment:
Can you fix this formatting so there are braces for the if and the else, with proper indentation?

Reply:
fixed